src/js/components/BreadcrumbComponent.jsx

```diff
@@ -1,7 +1,12 @@
+var _ = require("underscore");
+var classNames = require("classnames");
 var React = require("react/addons");
 var Link = require("react-router").Link;
 
 var PathUtil = require("../helpers/PathUtil");
+var StyleDimensions = require("../constants/StyleDimensions");
```

Comment:
It's again about fragmentation. Why generalize things, that don't need a generalization? That is unnecessary work done. Also for the next person that wonders why this is generalize and assumes that this is used somewhere else.